packages/graphql-transformers-e2e-tests/package.json

```diff
@@ -65,7 +65,6 @@
     "transform": {
       "^.+\\.tsx?$": "ts-jest"
     },
-    "testEnvironment": "node",
```

Comment:
Removing in favor of custom env on L116 below, which extends Jest's `NodeEnvironment`